.gitattributes

```diff
@@ -8,3 +8,4 @@
 *.hpp diff=cpp
 *.c++ diff=cpp
 *.h++ diff=cpp
+src/lib/uri/uri_parser.c linguist-generated=true
```

Comment:
I've pushed it in fbea9862b087e161a48dc0ac181a1da06cc0ce9d. Please, rebase this patchset.

Reply:
rebased